### PR TITLE
[Blog] Publish the Learning path blogpost

### LINKED ISSUE #2323: [Blog] Publish the Learning path blogpost
Contributors and community members are encouraged to post on https://layer5.io/blog. A new blog post fitting the following description is requested to be written by a community member:

**Desired Title:**  Introducing Layer5 Learning Paths
**Draft:**  https://docs.google.com/document/d/1qqGluGEePjcBl2R79M_6h6wfu81xYaFcMgeSPLNDKt4/edit?usp=sharing
**Author:** Anita Ihuman

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Anita-ihuman**
**Description**
Created a blog post for the Layer5 learning path
This PR fixes #2323

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/blog/2021-11-15-layer5-learning-path /index.mdx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+import { BlogWrapper } from "../Blog.style.js";
+
+<BlogWrapper>
+
+ <img style="max-height:155px; display: inline; position: relative; float: right; margin-left:0.25rem;" src={LearningPath} />
+ <h1>Introducing Layer5 Learning Paths</h1>
+<p>
+After five months of community-based development, the Layer5 <a href="https://layer5.io/learn/learning-paths">Learning Paths</a> project, is officially launched! Exciting as it sounds, in addition to the existing free <a href=” https://layer5.io/learn”>service mesh training resources</a> available to you, Layer5 Learning Paths offer curated guides to assist you in developing your general cloud native networking knowledge and service mesh-specific skills. 
+<p/>
```
Comment (Mr-DG-Wick) [inline src/collections/blog/2021-11-15-layer5-learning-path /index.mdx:29]:
```suggestion
</p>
```

File: src/collections/blog/2021-11-15-layer5-learning-path /index.mdx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+ <h3> 
+  Learning Paths information architecture
+</h3>
+ <img style="max-height:155px; display: inline; position: relative; float: right; margin-left:0.25rem;" src={Architecture} />
+<p>
+  Since this is the first phase of the Learning Path/Certification project, only
+  one learning path is currently available: Mastering Service Meshes For
+  Developers, which is two courses in length with ~8 chapters per course. The
+  courses in this learning path focus on the fundamentals of Service Meshes and
+  use <a href="https://meshery.io/">Meshery</a> as the management plane to manage the service meshes. It comprises a hands-on tutorial on <a href="https://layer5.io/learn/learning-paths/mastering-service-meshes-for-developers/introduction-to-service-meshes">introduction to Service Meshes</a> and an <a href="https://layer5.io/learn/learning-paths/mastering-service-meshes-for-developers/advance-concepts-of-service-mesh" introduction to the advanced concepts of Service Meshes</a>. Within each course, you will find
```
Comment (Mr-DG-Wick) [inline src/collections/blog/2021-11-15-layer5-learning-path /index.mdx:58]:
```suggestion
  use <a href="https://meshery.io/">Meshery</a> as the management plane to manage the service meshes. It comprises a hands-on tutorial on <a href="https://layer5.io/learn/learning-paths/mastering-service-meshes-for-developers/introduction-to-service-meshes">introduction to Service Meshes</a> and an <a href="https://layer5.io/learn/learning-paths/mastering-service-meshes-for-developers/advance-concepts-of-service-mesh"> introduction to the advanced concepts of Service Meshes</a>. Within each course, you will find
```

Comment (Mr-DG-Wick) [inline src/collections/blog/2021-11-15-layer5-learning-path /index.mdx:58]:
```suggestion
  use <a href="https://meshery.io/">Meshery</a> as the management plane to manage the service meshes. It comprises a hands-on tutorial on <a href="https://layer5.io/learn/learning-paths/mastering-service-meshes-for-developers/introduction-to-service-meshes">introduction to Service Meshes</a> and an <a href="https://layer5.io/learn/learning-paths/mastering-service-meshes-for-developers/advance-concepts-of-service-mesh"> introduction to the advanced concepts of Service Meshes</a>. Within each course, you will find
```

File: src/collections/blog/2021-11-15-layer5-learning-path /index.mdx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+redirect_from:
+  - /blog/community/layer5-learning-path
+---
+
+import { Link } from "gatsby";
+
+import BasicsofServiceMesh from "./mastering-service-meshes-for-Developers.png";
+import Architecture from "./course-architecture.jpg";
+import LearningPath from "./learning-path.jpg";
+import ServiceMesh from "./servicemesh.jpg";
```
Comment (Mr-DG-Wick) [inline src/collections/blog/2021-11-15-layer5-learning-path /index.mdx:19]:
```suggestion
import ServiceMesh from "./servicemeshes.png";
```

File: src/collections/blog/2021-11-15-layer5-learning-path /index.mdx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+  - Community
+published: true
+redirect_from:
+  - /blog/community/layer5-learning-path
+---
+
+import { Link } from "gatsby";
+
+import BasicsofServiceMesh from "./mastering-service-meshes-for-Developers.png";
+import Architecture from "./course-architecture.jpg";
```
Comment (Mr-DG-Wick) [inline src/collections/blog/2021-11-15-layer5-learning-path /index.mdx:17]:
```suggestion
import Architecture from "./course-architecture.png";
```

**Review by Mr-DG-Wick — CHANGES_REQUESTED** [review]
Thanks for taking this up @Anita-ihuman ! A good rule of thumb is to use .png image format over the .jpg format.

File: src/collections/blog/2021-11-15-layer5-learning-path /index.mdx [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+</p>
+<p>
+The Learning Path is an initiative developed to help enlighten and test your knowledge on service meshes and management of the workloads that you’ll build atop of them. As a light Learning Management System, Layer5 Learning Paths offer cloud native training and certification through the lens of role-based paths with interactive labs intertwined through the chapters. Oriented around your role as a developer, operator, or security engineer, you can choose your learning path most relevant to your everyday focus.
+</p>
+<p>
+Learning Paths comprise courses that are hand-curated by the maintainers at Layer5 and equip you with the foundational tools like <a href=”https://meshery.io/”>Meshery</a> that assists you in comprehending service mesh concepts. <a href=”https://layer5.io/learn/learning-paths/mastering-service-meshes-for-developers”>Mastering Service Meshes For Developers</a> is the initial Learning Path offering that focuses on Developers and what you need to know to successfully design workloads for a service mesh environment.
```
Comment (Mr-DG-Wick) [inline src/collections/blog/2021-11-15-layer5-learning-path /index.mdx:34]:
```suggestion
Learning Paths comprise courses that are hand-curated by the maintainers at Layer5 and equip you with the foundational tools like <a href="https://meshery.io/">Meshery</a> that assists you in comprehending service mesh concepts. <a href="https://layer5.io/learn/learning-paths/mastering-service-meshes-for-developers">Mastering Service Meshes For Developers</a> is the initial Learning Path offering that focuses on Developers and what you need to know to successfully design workloads for a service mesh environment. 
```

File: src/collections/blog/2021-11-15-layer5-learning-path /index.mdx [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+import ServiceMesh from "./servicemesh.jpg";
+
+import { BlogWrapper } from "../Blog.style.js";
+
+<BlogWrapper>
+
+ <img style="max-height:155px; display: inline; position: relative; float: right; margin-left:0.25rem;" src={LearningPath} />
+ <h1>Introducing Layer5 Learning Paths</h1>
+<p>
+After five months of community-based development, the Layer5 <a href="https://layer5.io/learn/learning-paths">Learning Paths</a> project, is officially launched! Exciting as it sounds, in addition to the existing free <a href=” https://layer5.io/learn”>service mesh training resources</a> available to you, Layer5 Learning Paths offer curated guides to assist you in developing your general cloud native networking knowledge and service mesh-specific skills.
```
Comment (Mr-DG-Wick) [inline src/collections/blog/2021-11-15-layer5-learning-path /index.mdx:28]:
```suggestion
After five months of community-based development, the Layer5 <a href="https://layer5.io/learn/learning-paths">Learning Paths</a> project, is officially launched! Exciting as it sounds, in addition to the existing free <a href="https://layer5.io/learn">service mesh training resources</a> available to you, Layer5 Learning Paths offer curated guides to assist you in developing your general cloud native networking knowledge and service mesh-specific skills. 
```

**Comment by warunicorn19** [comment]
@Anita-ihuman closing this PR and creating a new, the main problem is happening because the file name has a white space in it.

**Pull request closed without merging**